backend/hub/hub.go

```diff
@@ -52,26 +56,27 @@ type hub struct {
 	gameFactory gamefactory.GameFactory
 	gameRouter  game.GameRouter
 
-	reg registry.Registry
-}
-
-// hubClient should only store basic information about the client
-// and the send channel
-type hubClient struct {
-	send     (chan []byte)
-	nickname string
+	reg         registry.Registry
+	tokenSecret []byte
```

Comment:
Each hub when created will get a token password created dynamically. So these passwords hypothetically will not be sticking around for too long, usually just during the duration that a group plays the game for